### PR TITLE
[Snapfile] Use consistent quotation

### PR DESCRIPTION
**Pull request by kylef**
It was inconsistently using single and double quotation so I've made a PR to use single quotation always.

### CODE REVIEW

File: example/Snapfile [hunk 1]
```diff
@@ -38,5 +38,5 @@ end
 
 teardown_device do |language, device|
   puts "Finished with #{language} on #{device}"
-  system("./cleanup.sh")
-end
\ No newline at end of file
+  system(''./cleanup.sh')
```
Comment (KrauseFx) [inline example/Snapfile]: Ups `''`

**Comment by KrauseFx** [comment]
Thanks @kylef. Why did you decide to go for single quotations? I went with double in `fastlane`, since it allows the user to easily add `#{variable}` without having to change the quotation.

What do you think?

**Comment by kylef** [comment]
I used single quotes where applicable to be compatible with the default rubocop rules.

```
StringLiterals:
  # Valid values (denoting the preferred quote delimiters) are:
  # single_quotes, double_quotes
  EnforcedStyle: single_quotes
```

**Comment by kylef** [comment]
Let me know what you want to do, I can fix that issue or switch to double quotes.

**Comment by KrauseFx** [comment]
I think I'll go with `"` as it's easier for non-Ruby developers to add dynamic values without having to change the quote style :+1:

**Comment by KrauseFx** [comment]
I'm using the same style in the [fastlane](https://github.com/KrauseFx/fastlane) repo.

**Comment by KrauseFx** [comment]
Since I'm re-writing everything to use the new UI Tests, I'll close this issue for now, thanks @kylef :+1:

**Pull request closed without merging**